chcemvediet/apps/inforequests/tests/test_cron.py

```diff
@@ -325,76 +320,70 @@ def test_reminder_is_not_sent_if_last_action_does_not_have_obligee_deadline(self
+    def test_reminder_is_not_sent_if_last_action_snooze_was_already_missed_when_last_reminder_was_sent(self):
         timewarp.jump(local_datetime_from_local(u'2010-10-05 10:33:00'))
-        last = utc_datetime_from_local(u'2010-10-11 10:33:00')
+        last = utc_datetime_from_local(u'2010-10-14 10:33:00')
         inforequest, _, _ = self._create_inforequest_scenario(
                 u'clarification_request',
-                # deadline is missed at 2010-10-11
-                (u'clarification_response', dict(deadline=5, last_deadline_reminder=last)),
+                # snooze is missed at 2010-10-14
+                (u'clarification_response', dict(delivered_date=naive_date(u'2010-10-05'), last_deadline_reminder=last)),
                 )
 
         timewarp.jump(local_datetime_from_local(u'2010-10-20 10:33:00'))
         message_set = self._call_cron_job()
         self.assertFalse(message_set.exists())
 
-    @unittest.skip('FIXME')
-    def test_reminder_is_sent_if_last_action_deadline_was_not_missed_yet_when_last_reminder_was_sent(self):
+    def test_reminder_is_sent_if_last_action_snooze_was_not_missed_yet_when_last_reminder_was_sent(self):
         timewarp.jump(local_datetime_from_local(u'2010-10-05 10:33:00'))
         last = utc_datetime_from_local(u'2010-10-10 10:33:00')
         inforequest, _, _ = self._create_inforequest_scenario(
                 u'clarification_request',
-                # deadline is missed at 2010-10-11
-                (u'clarification_response', dict(deadline=5, last_deadline_reminder=last)),
+                # snooze is missed at 2010-10-14
+                (u'clarification_response', dict(delivered_date=naive_date(u'2010-10-05'), last_deadline_reminder=last)),
                 )
 
         timewarp.jump(local_datetime_from_local(u'2010-10-20 10:33:00'))
         message_set = self._call_cron_job()
         self.assertTrue(message_set.exists())
 
-    @unittest.skip('FIXME')
-    def test_reminder_is_sent_if_last_action_deadline_was_already_missed_when_last_reminder_was_sent_but_it_was_extended_later(self):
+    def test_reminder_is_sent_if_last_action_snooze_was_already_missed_when_last_reminder_was_sent_but_it_was_extended_later(self):
         timewarp.jump(local_datetime_from_local(u'2010-10-05 10:33:00'))
         last = utc_datetime_from_local(u'2010-10-12 10:33:00')
         inforequest, _, _ = self._create_inforequest_scenario(
                 u'clarification_request',
-                # deadline was missed at 2010-10-11, but then it was extended by 3 days; it will be
-                # missed at 2010-10-14 again.
-                (u'clarification_response', dict(deadline=5, extension=3, last_deadline_reminder=last)),
+                # snooze was missed at 2010-10-13, but then it was extended by 5 days; it will be
+                # missed at 2010-10-18 again.
+                (u'extension', dict(delivered_date=naive_date(u'2010-10-05'), extension=5, last_deadline_reminder=last)),
```

Comment:
Pod "but it was extended later" sa nemysli EXTENSION action vykonana instituciou, ale predlzenie snooze userom.

Cize tu by mala byt nadalej CLARIFICATION_RESPONSE akcia, ktorej deadline expiroval, jeden remainer uz bol poslany, user predlzil snooze, a uz aj tento predlzeny snooze expiroval.

Reply:
:ok: